src/components/index/IndexComponent.tsx

```diff
@@ -99,6 +103,11 @@ const Index = ({
       return result;
     }, initial);
     setFilteredRequests(nextFR);
+    if (process.env.REACT_APP_DEBUG && nextFR.old.length)
+      console.log(
+        `${nextFR.old.length} requests too old to show: `,
+        nextFR.old
+      );
```

Comment:
added this because you may see requests on datastructure but not know why they are not showing up in UI. this will give you a hint at least if that happens